packages/messaging/ios/RNFBMessaging/RNFBMessagingModule.m

```diff
@@ -320,6 +320,12 @@ - (NSDictionary *)constantsToExport {
               authorizedStatus = @2;
             }
           }
+        
+        if (@available(iOS 14.0, *)) {
```

Comment:
I see from the docs that macCatalyst can also do it (iPadOS is not an available `available` check, it collapses to iOS I guess?) and I've just worked to make sure catalyst is a vaild target, so I'd like to add it here

No action should be required for me to apply this suggestion if GitHub is working the way I think it is


```suggestion
        if (@available(iOS 14.0, macCatalyst 14.0, *)) {
```